analyzer/codechecker_analyzer/buildlog/build_action.py

```diff
@@ -73,7 +73,7 @@ def __hash__(self):
         hash_content = []
         hash_content.extend(self.analyzer_options)
         hash_content.append(str(self.analyzer_type))
-        hash_content.append(self.target)
+        hash_content.append(self.target[self.lang])
```

Comment:
Because a BuildAction object is build from a dictionary (in the constructor) and `__slots__` magic I did not found any easy way without further hacking to used properties.